app/scripts/modules/core/src/pipeline/executions/execution/ExecutionMarker.tsx

```diff
@@ -81,9 +81,24 @@ export class ExecutionMarker extends React.Component<IExecutionMarkerProps, IExe
     });
   };
 
+  private getStageOutputStatusCode = (): number => {
+    const { stage } = this.props;
+    return stage.masterStage.outputs.status?.code;
+  };
+
+  private getClassNameByStatusCode = (code: number): string => {
+    switch (code) {
+      case 2:
+        return 'success-with-diff';
+      default:
+        return '';
+    }
+  };
```

Comment:
@bbrina - This is all very specific to our use case and doesn't belong in this repo. What can we do to extend `ExecutionMarker` in a way that allows us to support this specific use case in our extension of Deck?